code/__DEFINES/mobs.dm

```diff
@@ -60,9 +60,17 @@
 #define DEFAULT_BODYPART_ICON_ORGANIC 'icons/mob/human_parts_greyscale.dmi'
 #define DEFAULT_BODYPART_ICON_ROBOTIC 'icons/mob/augmentation/augments.dmi'
 
+
 #define MONKEY_BODYPART "monkey"
 #define ALIEN_BODYPART "alien"
 #define LARVA_BODYPART "larva"
+
+
+#define NULL_BODY 0
```

Comment:
Remove this, you should be using NONE instead.

Reply:
fixed